packages/medusa/src/interfaces/payment-service.ts

```diff
@@ -0,0 +1,120 @@
+import { TransactionBaseService } from "./transaction-base-service"
+import {
+  Cart,
+  Customer,
+  Payment,
+  PaymentSession,
+  PaymentSessionStatus,
+} from "../models"
+import { PaymentService } from "medusa-interfaces"
+
+export type Data = Record<string, unknown>
```

Comment:
question: This type seems rather generic and not tied to payment service. What's the thinking behind renaming Record, which has been used elsewhere?

Reply:
ahah it was just to control the Record in one place if we change it, instead of updating all the record in the place